contrib/pylightning/lightning/lightning.py

```diff
@@ -464,18 +464,16 @@ def waitsendpay(self, payment_hash, timeout=None):
         }
         return self.call("waitsendpay", payload)
 
-    def pay(self, bolt11, msatoshi=None, description=None, riskfactor=None):
+    def pay(self, bolt11, msatoshi=None, label=None, riskfactor=None):
```

Comment:
I'd propose to just add both `label` and `description` and then check that only one is set when constructing the payload (assigning that one to `label`). We can also warn about `description` being phased out and remove it once we remove the first wave of deprecated APIs.

Reply:
Well, if we accept both, lightningd already does that check?

Reply:
Right :+1: